server/apiCalls/transformers/memos.test.js

```diff
@@ -13,10 +13,16 @@ const offering_SF1625_202121 = {
   courseCode: 'SF1625',
   offeringId: '1',
   period: 'P1',
+  courseRoundApplications: [
+    {
+      course_round_application_code: '1',
+    },
+  ],
 }
 const memo_SF1625_202121_base = (courseCode = 'SF1625') => ({
   courseCode,
   ladokRoundIds: ['1'],
```

Comment:
`ladokRoundIds ` has to be removed to be sure tests working properly

Reply:
Removed